src/styles/main.scss

```diff
@@ -0,0 +1,15 @@
+@import 'variables';
+@import 'blocks/page';
+@import 'blocks/calendar';
+
+html,
+body {
+  margin: 0;
+  padding: 0;
+  min-height: 100vh;
+  min-width: 100vh;
```

Comment:
The `min-width` property should be set to `100vw`, not `100vh`. The `vh` unit represents the viewport's height, while `vw` represents the viewport's width. This might cause unexpected layout issues on screens with different aspect ratios.